app/src/main/java/com/example/henzoshimada/feeltrip/ElasticSearchController.java

```diff
@@ -409,7 +445,7 @@ public GetUsernameTask(String username) { // must specify username and password
         private Double currentlat;
         private Double currentlon; // TODO: actually pass in the user's current lat and lon to these variables
 
-        public GetFilteredMoodsTask(String searchmode, String pastweek, String mostrecent, String emotion, String friendsonly){ // must pass this specific set of Strings in this order while constructing
+        public GetFilteredMoodsTask(String searchmode){ // must pass this specific set of Strings in this order while constructing
```

Comment:
We have a filter controller that holds all selected filters now, this eliminates the need to have a constructor that takes 5 parameters.